src/Controllers/MyRadio/login.php

```diff
@@ -16,7 +16,7 @@
  */
 
 if (isset($_SESSION['memberid'])) {
-    URLUtils::redirect(Config::$base_url);
+    URLUtils::redirectWithMessage("/", "", "It seems you were trying to login while still being logged in, if that is not what happened, please contact the Computing Team.");
```

Comment:
The JS (somewhat) rightfully assumes that if the API returns that it hasn't got valid credentials from a API call with JS myradio, then there must not be someone logged in. However, if the API is weird (like now), it will try to goto login whilst still actually logged in, so lands at the homepage.